### ISSUE TITLE
pgbackrest install problem

### ISSUE THREAD

**Issue by yusufbulentavci** [vitabaks/postgresql_cluster#90]
Hi,
First of all, thank you for the project. 
I want to setup a 4 node cluster; 3 for postgres and one for pgbackrest.
I am using vagrant, and the nodes are debian/buster.
The problem: pgbackrest installation fails.
I am attaching configuration and logs...
[ansible.cfg.txt](https://github.com/vitabaks/postgresql_cluster/files/5916772/ansible.cfg.txt)
[inventory.txt](https://github.com/vitabaks/postgresql_cluster/files/5916773/inventory.txt)
[Vagrantfile.txt](https://github.com/vitabaks/postgresql_cluster/files/5916778/Vagrantfile.txt)
[allout.log](https://github.com/vitabaks/postgresql_cluster/files/5916779/allout.log)
[main.yml.txt](https://github.com/vitabaks/postgresql_cluster/files/5916782/main.yml.txt)

**Comment by vitabaks** [comment]
> Failed to find required executable gpg

It looks like your pgbackrest repository host doesn't have the gpg binary (`gnupg`).
The current version of the playbook assumes that you already have a pgbackrest server configured.

We did not check for the gnupg package on the pgbackrest server.
I added this check in pgbackrest role (commit 323ab6e64ec18f0bc06cf0865a31bc6fb2d7cee4)


> vars/main.yml

the parameter `patroni_cluster_bootstrap_method: "pgbackrest"` is not suitable if you do not have pgbackrest backups yet.
Use "`initdb`" for a new deployment, and make sure you don't have pgbackrest specified in the patroni_create_replica_methods variable.

P.S.
The playbook does not currently configure a backup (although it can prepare the configuration), the task of this playbook is to deploy a сluster, including deploying from a (pre-existing) backup.
